apps/src/code-studio/appOptions.js

```diff
@@ -217,16 +217,15 @@
  * @property {string} level_source_id
  * @property {string} level_source_image_url
  */
-
```

Comment:
nit: did you mean to remove this line? looks like the other typedefs have a break between them

Reply:
Will add back